### PR TITLE
Skunk Spray Neutralizer

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
With the addition of Skunks in #73537 I was curious if tomato paste removed skunk spray.  It doesn't but this recipe really works.

#### Describe the solution
Adds the recipe I found for neutralizing skunk spray.

#### Describe alternatives you've considered
Not immediately adding additional content based on skunks being added to the game in the same hour as they were added.  Adding a Mephitidae mutation tree.

### CODE REVIEW

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -331,6 +331,29 @@
     "flags": [ "NO_INGEST", "WATER_DISSOLVE" ],
     "use_action": { "type": "heal", "disinfectant_power": 4, "bite": 0.95, "move_cost": 2000 }
   },
+  {
+    "id": "skunk_spray_neutralizing_solution",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": { "str_sp": "skunk spray neutralizing solution" },
+    "description": "A combination of hyrdogen peroxide, baking soda, and dish soap that works together to transform skunk spray thiocetates into odorless acids.",
```
Comment (XygenSS) [inline data/json/items/comestibles/med.json:339]:
```suggestion
    "description": "A combination of hydrogen peroxide, baking soda, and dish soap that works together to transform skunk spray thiocetates into odorless acids.",
```

**Comment by TheMurderUnicorn** [comment]
Nice, I had no idea how to handle removing the stink.

File: data/json/effects.json [hunk 2]
```diff
@@ -2094,6 +2094,16 @@
...
+  {
+    "type": "effect_type",
+    "id": "skunk_spray_neutralizer",
+    "name": [ "Skunk Spray Neutralized" ],
+    "description": [ "A solution that neutralizes skunk odors." ],
+    "rating": "good",
+    "apply_message": "As you apply the solution, the skunk odor dissipates.",
+    "max_duration": "2 minutes",
+    "removes_effects": [ "skunk_spray" ]
+  }
```
Comment (Maleclypse) [inline data/json/effects.json:2106]:
```suggestion
  },
```

File: data/json/effects.json [hunk 3]
```diff
@@ -2094,6 +2094,16 @@
       }
     ]
   },
+  {
+    "type": "effect_type",
+    "id": "skunk_spray_neutralizer",
+    "name": [ "Skunk Spray Neutralized" ],
+    "description": [ "A solution that neutralizes skunk odors." ],
```
Comment (Maleclypse) [inline data/json/effects.json:2101]:
```suggestion
    "desc": [ "A solution that neutralizes skunk odors." ],
```

File: data/json/items/comestibles/med.json [hunk 4]
```diff
@@ -331,6 +331,29 @@
...
+    "charges": 10,
+    "symbol": "~",
+    "color": "white",
+    "phase": "liquid",
+    "container": "bottle_plastic_small",
+    "flags": [ "NO_INGEST", "WATER_DISSOLVE" ],
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You cover yourself in the solution everywhere the spray touched your body.",
+      "move_cost": 2000,
```
Comment (Maleclypse) [inline data/json/items/comestibles/med.json:353]:
```suggestion
```

**Pull request merged**